tests/models/gpt2/test_gpt2.py

```diff
@@ -17,11 +17,20 @@ def test_gpt2(record_property):
     m = AutoModelForSequenceClassification.from_pretrained(
         model_name, torch_dtype=torch.bfloat16
     )
+    for param in m.parameters():
+        param.requires_grad = False
```

Comment:
why?

Reply:
I noticed GPT-2 is using backward ops like `torch.ops.aten.native_layer_norm_backward.default`. Both backward and forward version return a tuple of 3 outputs, but the backwards version will use all 3 instead of just the first one. So for simplicity, I wanted to disable backward ops since we haven't added support for them yet. I could remove this and test with default parameters. Or maybe parameterize this and test with and without grad.

Reply:
Such things ideally should be done by infra

Reply:
That's fair. Most of the current transformer models share a lot of commonalities, so that can be merged to maybe a pytest fixture to reduce duplicates. Since GPT-2 isn't passing at this point, I can remove these from this PR for now.

Reply:
This change to GPT-2 has been reverted.